pygus/gus/allometrics.py

```diff
@@ -224,3 +275,37 @@ def fit_parametric(dbh):
             return max(0, estimate)
 
         return fit_parametric
+
+    @staticmethod
+    def fit_linear(params):
+        """Static method that sets the constant of the linear function."""
+        B0 = params["B0"]
+        B1 = params["B1"]
+        DBHMin = params["DBHMin"]
+        DBHMax = params["DBHMax"]
+
+        def fit_lin(dbh):
+            # converting the dbh in cm into inche
+            dbh = min(DBHMax, max(DBHMin, 0.393700787 * dbh))
+            estimate = B0 + (B1 * dbh)
+            # converting into meters
+            return max(0, 0.3048 * estimate)
+
+        return fit_lin
+    
+    @staticmethod
+    def fit_log_linear(params):
+        """Static method that sets the constant of the linear function."""
+        B0 = params["B0"]
+        B1 = params["B1"]
+        DBHMin = params["DBHMin"]
+        DBHMax = params["DBHMax"]
+
+        def fit_log_lin(dbh):
+            # converting the dbh in cm into inche
+            dbh = min(DBHMax, max(DBHMin, 0.393700787 * dbh))
+            estimate = B0 + (B1 * np.log(dbh))
+            # converting into meters
+            return max(0, 0.3048 * estimate)
+
+        return fit_log_lin
```

Comment:
New equation types only relevant for enhanced allometrics file. But including them here anyway for future simplicity